docs/index.rst

```diff
@@ -0,0 +1,29 @@
+Welcome to plankton's documentation!
+====================================
+
+Plankton is a Python package that makes it easy to develop complex stateful device simulations. It
+is licensed under GPL version 3 and the source is available on github_, where you are welcome to
+open new issues so the package can improve.
+
+If you're new to Plankton reading the :ref:`user_guide` is a good start. Once you plan on writing
+your own device it's time to go through the :ref:`developer_guide`. The :ref:`api_documentation`
```

Comment:
I decided to link to the docs directly, but keep the labels for linking to this from elsewhere (in case the structure ever changes).